lib/index.ts

```diff
@@ -109,6 +109,30 @@ const sdk = (...args: Parameters<typeof createSdk>) => {
       /* istanbul ignore next */
       throw Error('could not validate token')
     },
+
+    async exchangeAccessKey(accessKey: string): Promise<AuthenticationInfo> {
+      let resp: SdkResponse<any>
+      try {
+        resp = await this.accessKey.exchange(accessKey)
+      } catch (error) {
+        logger?.error('failed to exchange access key', error)
+        throw Error('could not exchange access key')
+      }
+
+      const { sessionJwt } = resp.data
+      if (!sessionJwt) {
+        logger?.error('failed to parse exchange access key response')
+        throw Error('could not exchange access key')
```

Comment:
Why do we have different messages on log vs exception?
This lib is used in the server so this info is not exposed to clients, just curios